index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+            instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+            </dt>
+            <dd>
+              <p>
+                This array contains <a>AudioParamDescriptors</a>, which are
+                descriptors for each controllable parameter on an
+                <a>AudioNode</a>. This static property is referred in the
+                construction of an <a>AudioWorkletNode</a> to create instances
+                of <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter provides an object contains the information of the
+                associated <a>AudioContext</a> such as the sample rate and the
+                current playback time when queried.  
+              </p>
+            </dd>
+            <dt>
+              attribute EventHandler onmessage
+            </dt>
+            <dd>
+              <p>
+                The onmessage handler is called whenever the
+                <a>AudioWorkletNode</a> posts a node message back to the
+                corresponding processor object on the audio <a>rendering
+                thread</a>.  
+              </p>
+            </dd>
+            <dt>
+              void process()
+            </dt>
+            <dd>
+              <p>
+                This method gets executed isochronously by the audio rendering
+                thread along with the rest of an audio graph while the
+                associated <a>AudioWorkletNode</a> is connected.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  Float32Array[][] inputs
+                </dt>
+                <dd>
+                  Represents incoming data from inputs of the node. Each input
+                  can contain multiple channels.
+                </dd>
+                <dt>
+                  Float32Array[][] outputs
+                </dt>
+                <dd>
+                  Represents outgoing data from outputs of the node. Each output
+                  can contain multiple channels.
+                </dd>
+                <dt>
+                  Object parameters
+                </dt>
+                <dd>
+                  Represents an object contains multiple arrays of
+                  <a>AudioParam</a> data with the associated name as a key.
+                </dd>
+              </dl>
+            </dd>
+            <dt>
+              void postMessage()
+            </dt>
+            <dd>
+              <p>
+                It may be called to send a message to the
+                <a>AudioWorkletNode</a>, via the algorithm defined by the
+                Worker specification.
```

Comment:
It's probably easy to say that a postMessage from (main->audio) just queues up the message in a buffer.

The buffer is then emptied right before the `process` method call is invoked.

I'm assuming the actual timing of the postMessage event isn't important? (if it is can add a `time` attribute).